courses/free-courses-hi.md

```diff
@@ -419,6 +420,13 @@
 * [React Native Tutorials for Begginers](https://youtube.com/playlist?list=PL8kfZyp--gEXs4YsSLtB3KqDtdOFHMjWZ) - Programming with Mash
 
 
+#### Redux
+
+* [Redux toolkit crash course | Chai aur React Series](https://youtu.be/1i04-A7kfFI?si=_0I8vhyhK_5ISJEg) - Chai aur Code
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
no shortened links
remove trang (`si` ) parameters

Reply:
updated links